Altis_Life.Altis/stringtable.xml

```diff
@@ -1329,7 +1329,7 @@
             <Spanish>You must select a player you are sending the text to!</Spanish>
             <Russian>You must select a player you are sending the text to!</Russian>
             <German>Du musst einen Spieler auswählen!</German>
-            <French>Vous devez sélectionner un joueur à qui envoyer le message !</French>
+            <French>Vous devez sélectionner un joueur auquel vous envoyez le message !</French>
```

Comment:
Instead of making the sentence much longer, why not turning it in an easier way ?

Reply:
I think the sentence is more correct but it is debatable.

Reply:
What about : "Vous devez sélectionner un joueur auquel envoyer le message !"
One less "vous", same meaning.